server/supernova/sc/sc_plugin_interface.cpp

```diff
@@ -965,8 +977,12 @@ void sc_plugin_interface::buffer_write(uint32_t index, const char* filename, con
 
     if (leave_open && !buf->sndfile)
         buf->sndfile = sf.takeOwnership();
+#else
+    throw std::runtime_error("cannot write to buffer as supernova was compiled without libsndfile");
```

Comment:
Very pedantic nitpick: sometimes you start your error messages with an uppercase character, other times with a lowercase character. Try to be consistent.